lib/experimental/Information/Communities/Post/PostEvent/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { DateAvatar } from "@/experimental/Information/Avatars/DateAvatar"
+import { withSkeleton } from "@/lib/skeleton"
+import { Skeleton } from "@/ui/skeleton"
+
+type PostEventProps = {
+  title: string
+  image?: string
+  place: string
+  date: {
+    hour: string
+    day: number
+    month: string
+  }
+}
+
+export const BasePostEvent = ({
+  title,
+  image,
+  place,
+  date,
+}: PostEventProps) => {
+  return (
+    <div className="flex w-full flex-col gap-1 rounded-xl border border-solid border-f1-border-secondary bg-f1-background-inverse-secondary p-1 shadow">
+      {image && (
+        <img
```

Comment:
I think it is fair to consider images here as non-critical resource and apply [`loading=lazy`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#loading)